concepticondata/conceptlists/Bremer-2016-200.tsv

```diff
@@ -0,0 +1,201 @@
+Bremer-2016-200-83	83	farmer	979	FARMER
+Bremer-2016-200-84	84	sew, mend	1457	SEW
+Bremer-2016-200-85	85	yesterday	1174	YESTERDAY
+Bremer-2016-200-86	86	jump	1829	JUMP
+Bremer-2016-200-87	87	baboon	1197	BABOON
+Bremer-2016-200-88	88	be different, differ		
+Bremer-2016-200-89	89	day before yesterday	1180	DAY BEFORE YESTERDAY
+Bremer-2016-200-90	90	today	1283	TODAY
+Bremer-2016-200-91	91	grow	479	GROW
+Bremer-2016-200-92	92	be cold	1287	COLD
+Bremer-2016-200-93	93	ask	1309	ASK (INQUIRE)
+Bremer-2016-200-94	94	sit, sit down		
+Bremer-2016-200-95	95	be short	1645	SHORT
+Bremer-2016-200-96	96	smoke (n.)	778	SMOKE (EXHAUST)
+Bremer-2016-200-97	97	judge (v.)	1818	ADJUDICATE
+Bremer-2016-200-98	98	proud, conceited	174	PROUD
+Bremer-2016-200-99	99	oil	1484	OIL (ORGANIC SUBSTANCE)
+Bremer-2016-200-100	100	boat	1844	BOAT
+Bremer-2016-200-101	101	again	204	AGAIN
+Bremer-2016-200-102	102	money	1559	MONEY
+Bremer-2016-200-103	103	rabbit	1136	RABBIT
+Bremer-2016-200-104	104	before (time)	1882	BEFORE
+Bremer-2016-200-105	105	bamboo	1927	BAMBOO
+Bremer-2016-200-106	106	be green	1425	GREEN
+Bremer-2016-200-107	107	dew	1977	DEW
+Bremer-2016-200-108	108	greet, visit		
+Bremer-2016-200-109	109	one day	1260	DAY (24 HOURS)
+Bremer-2016-200-110	110	or	1014	OR
+Bremer-2016-200-111	111	fish	227	FISH
+Bremer-2016-200-112	112	riddle	2970	RIDDLE
+Bremer-2016-200-113	113	problem, disturbance	2425	PROBLEM
+Bremer-2016-200-114	114	small, little	1246	SMALL
+Bremer-2016-200-115	115	spine, back		
+Bremer-2016-200-116	116	behind, (in the) back of	1686	BEHIND
+Bremer-2016-200-117	117	tail	1220	TAIL
+Bremer-2016-200-118	118	shepherd	2544	SHEPHERD
+Bremer-2016-200-119	119	tree	906	TREE
+Bremer-2016-200-120	120	wood	1803	WOOD
+Bremer-2016-200-121	121	mourn, cry	1839	CRY
+Bremer-2016-200-122	122	be dry	1398	DRY
+Bremer-2016-200-123	123	egg	744	EGG
+Bremer-2016-200-124	124	moon	1313	MOON
+Bremer-2016-200-125	125	play	1413	PLAY
+Bremer-2016-200-126	126	star	1430	STAR
+Bremer-2016-200-127	127	week	1469	WEEK
+Bremer-2016-200-128	128	always, daily		
+Bremer-2016-200-129	129	blood	946	BLOOD
+Bremer-2016-200-130	130	be bitter	887	BITTER
+Bremer-2016-200-131	131	laugh	1355	LAUGH
+Bremer-2016-200-132	132	eat, chew meat		
+Bremer-2016-200-133	133	whistle	1025	WHISTLE
+Bremer-2016-200-134	134	dream	2455	DREAMING OR DREAM
+Bremer-2016-200-135	135	village	930	VILLAGE
+Bremer-2016-200-136	136	person	683	PERSON
+Bremer-2016-200-137	137	dark yellow		
+Bremer-2016-200-138	138	sleep (v.)	1585	SLEEP
+Bremer-2016-200-139	139	claw, fingernail	2128	CLAW OR NAIL
+Bremer-2016-200-140	140	gift	2982	GIFT
+Bremer-2016-200-141	141	ours	780	OUR
+Bremer-2016-200-142	142	mine	2090	MY
+Bremer-2016-200-143	143	yours (sg)	732	YOUR (SINGULAR)
+Bremer-2016-200-144	144	lock (n.)	1596	LOCK
+Bremer-2016-200-145	145	sun	1343	SUN
+Bremer-2016-200-146	146	die (v.)	1494	DIE
+Bremer-2016-200-147	147	why?	1270	WHY
+Bremer-2016-200-148	148	be pregnant	1123	PREGNANT
+Bremer-2016-200-149	149	bleed		
```

Comment:
I remember that bleed was proposed in the under-review list, right? So I suppose to already map it here.